app/services/amr/csv_parser.rb

```diff
@@ -0,0 +1,45 @@
+require 'csv'
+
```

Comment:
This just parses, including dropping the header row (if present) and doing the shift by one for Frome if appropriate. The output being a 'clean' array of rows for the importer to handle